src/foam/nanos/dig/DIG.js

```diff
@@ -128,33 +145,18 @@ foam.CLASS({
           query = true;
           url += "id=" + key;
         }
-        if ( email ) {
-          url += query ? "&" : "?";
-          query = true;
-          url += "email=" + email;
-
-          if ( subject ) {
-            url += "&";
-            url += "subject=" + encodeURIComponent(subject);
-          }
-        }
         if ( q ) {
           url += query ? "&" : "?";
           query = true;
           url += "q=" + encodeURIComponent(q);
         }
-        if ( limit >= 0 && limit != Number.MAX_SAFE_INTEGER ) {
+        if ( limit > 0 && limit != Number.MAX_SAFE_INTEGER ) {
```

Comment:
If they don't submit the limit in the url connection, what value will limit be?

Reply:
unlimited.